app/forms/provider_interface/offer_wizard.rb

```diff
@@ -116,6 +123,19 @@ def state
       as_json(except: %w[state_store errors validation_context course_option wizard_path_history]).to_json
     end
 
+    def setup_further_conditions(attrs)
+      return unless attrs['further_conditions']
+
+      attrs['further_conditions'].each do |index, value|
+        singleton_class.class_eval do
+          attr_accessor "further_conditions_#{index}"
+          alias :"further_conditions[#{index}]" :"further_conditions_#{index}"
+          alias :"further_conditions[#{index}]=" :"further_conditions_#{index}="
```

Comment:
The ideal here would be to have an attr_accessor for `further_conditions[index]`, but that isn't allowed because `[]` aren't allowed in variable names.

It's required to let the model populate values in the form, and to correctly link up the errors